src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -276,20 +276,27 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
         <div id="messagesContainer">
           <%
             for (Message message : messages) {
-              String author = UserStore.getInstance().getUser(message.getAuthorId()).getName();
+              User authorUser = UserStore.getInstance().getUser(message.getAuthorId());
+              String author = authorUser.getName();
+              String picture = authorUser.getPicture();
+
+              if(picture == null){
+                picture = "/default-user.png";
+              }
+
               if(user.equals(author)){
           %>
                 <div class="messageContainer right">
                   <div class="box triangle rightBubble blue">
                     <%= message.getContent() %>
                   </div>
-                  <img class="dot" src="/default-user.png">
+                  <img class="dot" src="<%= picture %>">
```

Comment:
any chance we can add a link on the image so the logged in user can access his/her profile from the chat page

Reply:
Sure, I can try and do that.